starters/e2e/e2e.streaming.e2e.ts

```diff
@@ -53,6 +52,7 @@ test.describe("streaming", () => {
     await expect(count).toHaveText("Rerender: 1");
 
     await rerender.click();
+    await page.waitForTimeout(5_000);
```

Comment:
@Varixo does this mean that we are adding a 5 seconds delay? 

Can we instead wait for an expected element / text to show up or something?

Reply:
we need to wait for the resources, I will look at it if we can do this better

Reply:
I removed this, after lowering the waiting time this is not needed